src-tool/Pact/Analyze/Check.hs

```diff
@@ -627,29 +627,34 @@ moduleCapabilities
   :: DynEnv -> [ModuleData Ref] -> ExceptT VerificationFailure IO [Capability]
 moduleCapabilities de mds = fmap concat $ forM mds $ \md -> do
```

Comment:
This function used to ignore the metadata creating a new `Capability` via `mkCap`. Now it adds `DefMeta` as a parameter to `mkCap`, so that the `Capability`'s `CapabilityAnnon` may be update accordingly.

Does the docstring for `moduleCapabilities` need to be updated to mention this? Or is the commend still correct, because your changes to `moduleCapabilities` just addressed a previous oversight?